### PR TITLE
feat: reduce severity of phishing filter to allow url token names

### PR DESCRIPTION
**Pull request by JFrankfurt**
this is a lower-complexity version of #6249

https://uniswaplabs.atlassian.net/browse/WEB-2802?atlOrigin=eyJpIjoiMDM1OGMyZTg4ODMzNGFhMTk2NTQ4YjhjZTg1Y2UxNTkiLCJwIjoiaiJ9

test by walletconnecting with 0x11E4857Bb9993a50c685A79AFad4E6F65D518DDa and viewing LP positions on Optimism

### CODE REVIEW

**Review by cmcewen — COMMENTED** [review]
this looks good to me but codecov would like some more coverage 🙏

**Review by tinaszheng — APPROVED** [review]

File: src/utils/urlChecks.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export function hasURL(str?: string): boolean {
+export function checkStringForURL(str?: string): boolean {
```
Comment (tinaszheng) [inline src/utils/urlChecks.ts:1]: i kind of prefer the `hasURL` version tbh, it's more clear that it returns a boolean

this kind of feels like it's more of a validation function

Comment (JFrankfurt) [reply src/utils/urlChecks.ts:1]: ok, @tinaszheng gets `hasURL` and @cmcewen gets `shouldHidePosition`

deal? 😅

Comment (cmcewen) [reply src/utils/urlChecks.ts]: Deal 🤝

**Review by cmcewen — APPROVED** [review]

**Pull request merged**